### DISCUSSION TITLE
Release >2.2.5 Breaks Height of Editor

### DISCUSSION THREAD

**Discussion by richdynamix** [awcodes/filament-tiptap-editor#95] [Q&A]
I just noticed that when we use the TipTap editor in full screen mode the height of the content area is cut off. I couldn't see any changes that would affect this but it seems to be related to version 2.2.5 as downgrading to 2.2.4 fixes the issue.

**Full screen editing in 2.2.4**
<img width="2560" alt="Screenshot 2023-03-17 at 17 26 07" src="https://user-images.githubusercontent.com/1104549/225976143-536df46d-28d5-46e7-a56e-6d891a1540cd.png">

**Full screen editing in >=2.2.5**
<img width="2560" alt="Screenshot 2023-03-17 at 17 26 30" src="https://user-images.githubusercontent.com/1104549/225976247-9a2a46f1-765d-4e03-99ab-9fb14a69cbe8.png">

Is there a configuration we should be looking for to resolve this?

**Comment by awcodes** [comment]
v2.2.7 fixes this. thanks for reporting.
